deployments/salt/splunk-otel-collector/collector_config.sls

```diff
@@ -0,0 +1,50 @@
+{% set splunk_service_user = salt['pillar.get']('splunk-otel-collector:service_user', 'splunk-otel-collector') %}
+
+{% set splunk_service_group = salt['pillar.get']('splunk-otel-collector:service_group', 'splunk-otel-collector') %}
+
+{% set splunk_access_token = salt['pillar.get']('splunk-otel-collector:splunk_access_token') %}
+
+{% set splunk_realm = salt['pillar.get']('splunk-otel-collector:splunk_realm', 'us0') %}
+
+{% set splunk_api_url = salt['pillar.get']('splunk-otel-collector:splunk_api_url', 'https://api.' + splunk_realm + '.signalfx.com' ) %}
+
+{% set splunk_ingest_url = salt['pillar.get']('splunk-otel-collector:splunk_ingest_url', 'https://ingest.' + splunk_realm + '.signalfx.com') %}
+
+{% set splunk_trace_url = salt['pillar.get']('splunk-otel-collector:splunk_trace_url', splunk_ingest_url + '/v2/trace') %}
+
+{% set splunk_hec_url = salt['pillar.get']('splunk-otel-collector:splunk_hec_url', splunk_ingest_url + '/v1/log') %}
+
+{% set splunk_hec_token = salt['pillar.get']('splunk-otel-collector:splunk_hec_token', splunk_access_token) %}
+
+{% set splunk_otel_collector_config = salt['pillar.get']('splunk-otel-collector:splunk_otel_collector_config', '/etc/otel/collector/agent_config.yaml') %}
+
+{% set splunk_collectd_dir = salt['pillar.get']('splunk-otel-collector:splunk_collectd_dir', '/usr/lib/splunk-otel-collector/agent-bundle/run/collectd') %}
+
+{% set splunk_bundle_dir = salt['pillar.get']('splunk-otel-collector:splunk_bundle_dir', '/usr/lib/splunk-otel-collector/agent-bundle') %}
+
+{% set splunk_memory_total_mib = salt['pillar.get']('splunk-otel-collector:splunk_memory_total_mib', '512') %}
+
+{% set splunk_ballast_size_mib = salt['pillar.get']('splunk-otel-collector:splunk_ballast_size_mib', '') %}
+
+/etc/otel/collector/splunk-otel-collector.conf:
+  file.managed:
```

Comment:
Please make sure the collector service is restarted if this environment file has changed.

Reply:
Thanks for the suggestions.
Updated with the suggested changes.